src/pint/observatory/topo_obs.py

```diff
@@ -509,3 +541,58 @@ def export_all_clock_files(directory):
         if hasattr(o, "_clock") and o._clock is not None:
             for clock in o._clock:
                 clock.export(directory / Path(clock.filename).name)
+
+
+def load_observatories(filename=observatories_json, overwrite=False):
+    """Load observatory definitions from JSON and create :class:`pint.observatory.topo_obs.TopoObs` objects, registering them
+
+    Set `overwrite` to ``True`` if you want to re-read a file with updated definitions.
+    If `overwrite` is ``False`` and you attempt to add an existing observatory, an exception is raised.
+
+    Parameters
+    ----------
+    filename : str or file-like object, optional
+    overwrite : bool, optional
+        Whether a new instance of an existing observatory should overwrite the existing one.
+
+    Raises
+    ------
+    ValueError
+        If an attempt is made to add an existing observatory with ``overwrite=False``
+
+    Notes
+    -----
+    If the ``origin`` field is a list of strings, they will be joined together with newlines.
+    """
+    # read in the JSON file
+    with open_or_use(filename, "r") as f:
+        observatories = json.load(f)
+
+    for obsname, obsdict in observatories.items():
+        if "origin" in obsdict:
+            if isinstance(obsdict["origin"], list):
+                obsdict["origin"] = "\n".join(obsdict["origin"])
+        if overwrite:
+            obsdict["overwrite"] = True
+        # create the object, which will also register it
+        TopoObs(name=obsname, **obsdict)
+
+
+def load_observatories_from_usual_locations(clear=False):
+    """Load observatories from the default JSON file as well as ``$PINT_OBS_OVERRIDE``, optionally clearing the registry
+
+    Parameters
```

Comment:
Maybe worth clarifying that with `clear=True` this returns PINT to the state it normally is on import? Modulo changes in the environment variables.

Reply:
clarified.